rasa/nlu/cli/server.py

```diff
@@ -22,6 +22,10 @@ def add_server_arguments(parser):
                              "provide this token as a query parameter")
     parser.add_argument('-w', '--write',
                         help='file where logs will be saved')
+    parser.add_argument('--loglevel',
```

Comment:
@eric-bonfadini We can remove this one then, right?

Reply:
Yes, right.